api/app/src/Controller/SelfRegisterController.php

```diff
@@ -109,7 +109,15 @@ public function verifyCoDeputy(Request $request, UserRegistrationService $userRe
 
         try {
             $coDeputyVerified = $userRegistrationService->validateCoDeputy($selfRegisterData);
-            $coDeputyUid = $userRegistrationService->retrieveCoDeputyUid();
+            $coDeputyUid = $userRegistrationService->retrieveCoDeputyUid($selfRegisterData);
+
+            $user = $this->em->getRepository('App\Entity\User')->findOneByEmail($selfRegisterData->getEmail());
+            if (!$user) {
+                throw new \RuntimeException('User registration: not found', 421);
+            }
+
```

Comment:
Do we need to check for the user here? It seems the user gets created in an earlier step.

Reply:
That was my error, I was supposed to remove this